games-fps/serioussam-tfe-data/serioussam-tfe-data-1.5-r1.ebuild

```
@@ -0,0 +1,62 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cdrom unpacker
+
+PATCH_PREFIX="Patch_1.05_FE"
+# Game name
+GN="serioussam"
+
+DESCRIPTION="Croteam's Serious Sam Classic The First Encounter ... the data files"
+HOMEPAGE="https://www.croteam.com/
+	https://store.steampowered.com/app/41050/Serious_Sam_Classic_The_First_Encounter/"
+SRC_URI="https://github.com/tx00100xt/serioussam-mods/raw/main/Patches/${PATCH_PREFIX}.tar.xz"
+
+S="${WORKDIR}"
```

This is supposed to be a single variables block, so no blank line between `SRC_URI` and `S` please:
```suggestion
DESCRIPTION="Croteam's Serious Sam Classic The First Encounter ... the data files"
HOMEPAGE="https://www.croteam.com/
	https://store.steampowered.com/app/41050/Serious_Sam_Classic_The_First_Encounter/"
SRC_URI="https://github.com/tx00100xt/serioussam-mods/raw/main/Patches/${PATCH_PREFIX}.tar.xz"
S="${WORKDIR}"
```